app-games/src/main/java/com/aptoide/android/aptoidegames/notifications/NotificationsPermissionRequester.kt

```diff
@@ -81,52 +83,80 @@ fun NotificationPermissionDialog(
       genericAnalytics.sendGetNotifiedContinueClick()
       onDismissDialog(true)
     }
-    DialogContent(onContinueClick)
+    val onCancelClick: () -> Unit = { onDismissDialog(false) }
+    DialogContent(onContinueClick, onCancelClick)
   }
 }
 
 @Preview
 @Composable
 fun DialogContentPreview() {
-  DialogContent {}
+  DialogContent({}, {})
 }
 
 @Composable
-fun DialogContent(onContinueClick: () -> Unit) {
+fun DialogContent(
+  onContinueClick: () -> Unit,
+  onCancelClick: () -> Unit
+) {
   Box(
     modifier = Modifier
       .padding(horizontal = 24.dp)
       .fillMaxWidth()
       .wrapContentHeight()
       .defaultMinSize(minHeight = 261.dp)
-      .paint(
-        painter = painterResource(id = R.drawable.notification_permission_bg),
-        contentScale = ContentScale.Crop,
-      ),
+      .background(color = Palette.Black),
     contentAlignment = Alignment.Center
   ) {
     Column(
       verticalArrangement = Arrangement.SpaceEvenly,
       modifier = Modifier
         .wrapContentHeight()
+        .padding(horizontal = 24.dp)
     ) {
+      Image(
+        imageVector = getNotificationsPermissionIcon(
+          controllerColor = Palette.Primary,
+          bellColor = Palette.White,
+          notificationColor = Palette.Secondary
+        ),
+        contentDescription = null,
+        modifier = Modifier
+          .fillMaxWidth()
+          .padding(vertical = 40.dp)
+      )
       Text(
         text = stringResource(R.string.notifications_context_title),
         style = AGTypography.Title,
         color = Palette.White,
-        modifier = Modifier.padding(top = 35.dp, start = 24.dp, bottom = 16.dp, end = 24.dp)
+        modifier = Modifier
+          .padding(bottom = 8.dp)
+          .align(Alignment.CenterHorizontally)
```

Comment:
You can use the Column horizontal alignment property instead of aligning each child